### DISCUSSION TITLE
How to align Profile Pic?

### DISCUSSION THREAD

**Discussion by awsaf49** [alshedivat/al-folio#1574] [Q&A] [answered]
It seems the profile pic is "down" aligned which sometimes creates unwanted empty space. I want to align the image in the "middle" (of up & down) but can't seem to figure it out. Your help is appreciated.

my website: https://awsaf49.github.io/

**Comment by awsaf49** [comment]
cc: @alshedivat

**Comment by george-gca** [comment, marked as answer]
If what you want to do is something similar to what I did [here](https://george-gca.github.io/), it is not an alignment problem, it is a hierarchy one. See, the profile picture is floating inside the [article tag](https://github.com/alshedivat/al-folio/blob/50f5620cd59c49d308b796717b659106876a5858/_layouts/about.html#L15C24-L15C24). If you want it to consider the header space, change its location in the html. In my case, I moved it to [inside header](https://github.com/george-gca/george-gca.github.io/blob/029c40ba8b917a74a98ee3ad878154ae096bb71b/_layouts/about.html#L10).

**Comment by awsaf49** [comment]
@george-gca Thanks, it solves my query
